content/guides/references/configuration.md

```diff
@@ -769,7 +780,7 @@ See the [plugins guide](/guides/tooling/plugins-guide) for more information.
 
 ## Common problems
 
-#### <Icon name="angle-right"></Icon> `baseUrl` is not set
+#### <Icon name="angle-right"></Icon> `baseUrl`&nbsp;is not set
```

Comment:
A search through the docs `.md` files shows that there are at least 170 instances like this throughout the docs, which should be fixed with CSS, not `&nbsp;`. Please revert this change.

<details>
  <summary>click to show the file list</summary>
  
```
❯ rg '^#.+`[^`]+` ' -g '*.md'
content/guides/getting-started/writing-your-first-component-test.md
234:### Using the `it()` function
566:### The `describe()` function

content/guides/guides/debugging.md
189:#### When clicking on `.type()` command, the Developer Tools console outputs the following:

content/guides/core-concepts/retry-ability.md
505:### Use `.should()` with a callback

content/guides/dashboard/projects.md
100:#### The `projectId` is a 6 character string in your Cypress configuration

content/guides/core-concepts/introduction-to-cypress.md
942:#### You cannot add a `.catch` error handler to a failed command

content/guides/guides/web-security.md
389:### Set `chromeWebSecurity` to `false`
405:#### Set `chromeWebSecurity` to `false` in the [Cypress configuration](/guides/references/configuration)

content/guides/continuous-integration/circleci.md
101:#### Example `.circleci/config.yml` v2 config file
132:#### Example `.circleci/config.yml` v2 config file with `yarn`

content/api/utilities/blob.md
55:#### Create an `img` element and set its `src` to the `dataUrl`

content/api/commands/as.md
132:### `as` is asynchronous

content/api/commands/and.md
230:### What's the difference between `.then()` and `.should()`/`.and()`?

content/api/commands/getcookie.md
58:#### Get `session_id` cookie after logging in
108:#### Using `cy.getCookie()` to test logging in

content/api/commands/clearlocalstorage.md
70:#### Clear all localStorage matching `/app-/` RegExp

content/guides/references/configuration.md
772:#### <Icon name="angle-right"></Icon> `baseUrl` is not set
812:#### <Icon name="angle-right"></Icon> Test files not found when using `spec` parameter

content/api/commands/trigger.md
84:#### Trigger a `mouseover` on the button
101:#### Trigger a `mousedown` from a specific mouse button
153:#### Trigger a `mousedown` on the top right of a button
178:#### Specify the exact `clientX` and `clientY` the event should have

content/api/commands/contains.md
88:#### Find the `input[type='submit']` by value
265:### Leading, trailing, duplicate whitespaces aren't ignored in `<pre>` tag
360:#### Favor of `<button>` over other deeper elements
379:#### Favor of `<a>` over other deeper elements
400:#### Favor of `<label>` over other deeper elements

content/api/commands/clock.md
193:#### `Using cy.clock()` with `cy.tick()`

content/api/commands/spy.md
73:#### More `cy.spy()` examples

content/api/commands/route.md
117:#### Wait on XHR `GET` request matching `url`
126:#### Wait on XHR's matching `method` and `url`
136:#### Setup route to `POST` to login
144:#### Wait on `url` matching glob
177:#### Override `url` glob matching options
199:#### `url` as a string
215:#### `url` as a RegExp
400:#### Setup route to error on `POST` to login
505:### `cy.route()` cannot be debugged using [`cy.request()`](/api/commands/request)
507:#### `cy.request()` sends requests to actual endpoints, bypassing those defined using `cy.route()`

content/api/commands/session.md
456:### Where to call `cy.mount()` <ComponentOnlyBadge />
550:### Where to call `cy.visit()` <E2EOnlyBadge />
880:#### Why am I seeing `401` errors after calling `cy.session()`?

content/api/commands/wait.md
65:#### When given a `time` argument:
70:#### When given an `alias` argument:
237:#### `requestTimeout` and `responseTimeout`

content/api/commands/writefile.md
84:#### Write some text to a `txt` file

content/api/commands/type.md
256:#### Type literal `{` or `}` characters
455:#### Preventing `mousedown` does not prevent typing
484:#### Typing `tab` key does not work

content/api/commands/nextuntil.md
72:#### Find all of the element's siblings following `#veggies` until `#nuts`

content/api/commands/request.md
143:#### Make a `GET` request
166:#### Send a `DELETE` request
186:#### Send a `POST` request with a JSON body
277:#### Using `cy.request()` for HTML Forms
287:#### Call `cy.request()` over and over again
360:#### `cy.request()` sends requests to actual endpoints, bypassing those defined using `cy.route()` or `cy.intercept()`

content/api/commands/server.md
310:#### `cy.server()` does not effect [cy.request()](/api/commands/request)

content/api/commands/fixture.md
81:#### Load a `users.json` fixture
131:#### Image fixtures are sent as `base64` by default
165:#### Using `.then()` to access fixture data
214:#### Using the `fixture` `StaticResponse` property
256:### `this` context

content/api/commands/visit.md
156:#### Provide an `onBeforeLoad` callback function
180:#### Provide an `onLoad` callback function
284:##### Visit local file when `baseUrl` is set
341:#### Visit will always yield the remote page's `window` object when it resolves

content/api/commands/intercept.md
409:#### With a `StaticResponse` object
1106:#### `req.reply()` shorthand
1279:#### `res.send()` shorthand
1317:## `StaticResponse` objects
1500:## `cy.intercept()` and request caching

content/api/commands/parentsuntil.md
72:#### Find all of the `.active` element's ancestors until `.nav`

content/api/commands/selectfile.md
99:#### `select` (default)

content/api/commands/hover.md
49:#### Simulating `mouseover` event to get popover to display

content/api/commands/mount.md
36:## Creating a New `cy.mount()` Command
113:## Adding TypeScript Typings for `cy.mount()` Commands

content/api/commands/parent.md
87:#### Get the parent with class `sub-nav` of all `li` elements

content/api/commands/select.md
61:#### Select the `option` with the text `apples`
78:#### Select the `option` with the value "456"
95:#### Select the `option` with index 0

content/api/commands/get.md
82:#### Find the first `li` descendent within a `ul`
132:#### `cy.get()` in the [`.within()`](/api/commands/within) command

content/api/commands/should.md
420:### What's the difference between `.then()` and `.should()`/`.and()`?

content/api/commands/screenshot.md
122:#### Take a screenshot of the first `.post` element
128:#### Take a screenshot of the first `.post` element with 10px of padding around it
140:### Get screenshot info from the `onAfterScreenshot` callback
230:### `after:screenshot` plugin event

content/api/commands/url.md
60:### `decode` option

content/api/commands/closest.md
60:#### Find the closest element of the `.error` with the class 'banner'
89:#### Find the closest element of `li.active` with the class 'nav'

content/api/commands/stub.md
156:#### More `cy.stub()` examples

content/api/commands/its.md
76:#### Get the `length` property of a DOM element
86:#### Get the `user` object of the response's `body`
106:#### Get `length` of title
150:#### Use `.its()` to test `window.fetch`

content/api/commands/parents.md
88:#### Get the parents with class `main-nav` of the active li

content/api/commands/invoke.md
74:#### Use `.invoke()` to test HTML content
96:#### Use `.invoke('show')` and `.invoke('trigger')`
119:#### Use `cy.invoke('removeAttr', 'target')` to get around new tab

content/api/commands/readfile.md
70:#### Read a `.txt` file

content/api/commands/hash.md
57:#### Assert that hash is `#/users/1` given remote URL: `http://localhost:8000/app/#/users/1`

content/api/commands/then.md
81:#### The `button` element is yielded
138:#### Returning `undefined` will not modify the yielded subject
217:### What's the difference between `.then()` and `.should()`/`.and()`?

content/api/cypress-api/cookies.md
161:#### `preserve` accepts:

content/api/commands/viewport.md
192:#### `devicePixelRatio` is not simulated

content/api/commands/blur.md
96:#### `.blur()` can time out because your browser did not receive any blur events

content/api/commands/tick.md
84:#### Using `cy.clock()` with `cy.tick()`

content/api/commands/prevuntil.md
72:#### Find all of the element's siblings before `#nuts` until `#veggies`

content/api/commands/pause.md
83:### Pause and step through each `.click()` command

content/api/cypress-api/browser.md
32:#### `Cypress.browser` returns browser object

content/guides/references/error-messages.md
121:### <Icon name="exclamation-triangle" color="red"></Icon> `cy...()` failed because the element you are chaining off of has become detached or removed from the dom
236:### <Icon name="exclamation-triangle" color="red"></Icon> `cy....()` failed because the element cannot be interacted with
266:### <Icon name="exclamation-triangle" color="red"></Icon> `cy....()` failed because the element is currently animating
446:### <Icon name="exclamation-triangle" color="red"></Icon> `cy.visit()` failed because you are attempting to visit a second unique domain
450:### <Icon name="exclamation-triangle" color="red"></Icon> `cy.visit()` failed because you are attempting to visit a different origin domain
465:### <Icon name="exclamation-triangle" color="red"></Icon> `Cypress.addParentCommand()` / `Cypress.addDualCommand()` / `Cypress.addChildCommand()` has been removed and replaced by `Cypress.Commands.add()`
477:### <Icon name="exclamation-triangle" color="red"></Icon> Cypress detected that you invoked one or more `cy` commands in a custom command but returned a different value.
489:### <Icon name="exclamation-triangle" color="red"></Icon> Cypress detected that you invoked one or more `cy` commands but returned a different value.
513:### <Icon name="exclamation-triangle" color="red"></Icon> Cypress detected that you returned a promise in a test, but also invoked one or more `cy` commands inside of that promise.
527:### <Icon name="exclamation-triangle" color="red"></Icon> Passing `cy.route({stub: false})` or `cy.server({stub: false})` is now deprecated.
542:### <Icon name="exclamation-triangle" color="red"></Icon> You passed the `--record` flag but did not provide us your Record Key.
558:### <Icon name="exclamation-triangle" color="red"></Icon> The `cypress ci` command has been deprecated
598:### <Icon name="exclamation-triangle" color="red"></Icon> Incorrect usage of `--ci-build-id` flag
612:### <Icon name="exclamation-triangle" color="red"></Icon> The `--ci-build-id`, `--group`, or `--parallel` flags can only be used when recording
694:### <Icon name="exclamation-triangle" color="red"></Icon> Run must pass `--parallel` flag

content/api/cypress-api/screenshot-api.md
102:### Scale `viewport` and `fullPage` captures
116:### Change the DOM using `onBeforeScreenshot` and `onAfterScreenshot`
147:### Get properties from the `onAfterScreenshot` callback

content/api/commands/location.md
58:#### When given a `key` argument:

content/api/cypress-api/env.md
217:### Why is it `Cypress.env` and not `cy.env`?

content/api/cypress-api/config.md
179:### `Cypress.config()` executes Synchronously
196:### Why is it `Cypress.config` and not `cy.config`?

content/guides/references/migration-guide.md
91:## Migrating from `cypress-file-upload` to [`.selectFile()`](/api/commands/selectfile)
279:### `cypress run` runs all browsers `--headless`
450:#### Deprecated `cy.route2()` command removed
469:#### `res.delay()` and `res.throttle()` have been renamed
500:#### Falsy values are no longer dropped in `StaticResponse` bodies
545:#### 1. Remove `experimentalComponentTesting` config
942:#### Legacy `cypress-react-unit-test` and `cypress-vue-unit-test` packages
1000:## Migrating `cy.route()` to [`cy.intercept()`][intercept]
1033:### Match against `url` and `path`
1068:### `cy.wait()` object
1217:#### Assert visibility of `opacity: 0` element
1241:#### Perform actions on `opacity: 0` element
1256:### `cy.wait(alias)` type
1551:### Cookies `whitelist` option renamed
1572:### `blacklistHosts` configuration renamed
1598:### Return type of `Cypress.Blob` changed
1623:### `cy.server()` `whitelist` option renamed
1648:### Cookies `sameSite` property
1776:#### <Icon name="exclamation-triangle" color="red"></Icon> Breaking Change: invoke `done` callback and return a promise
1907:#### <Icon name="exclamation-triangle" color="red"></Icon> Breaking Change: `empty` assertions
1928:#### <Icon name="exclamation-triangle" color="red"></Icon> Breaking Change: `include` checks strict equality
1979:#### <Icon name="exclamation-triangle" color="red"></Icon> Breaking Change: `reset()` replaced by `resetHistory()`
2173:### `cy.writeFile()` yields `null`

content/guides/references/best-practices.md
618:## Using `after` or `afterEach` hooks
849:### Unnecessary wait for `cy.visit()` <E2EOnlyBadge />
983:### Without `baseUrl` set, Cypress loads main window in `localhost` + random port
1006:### With `baseUrl` set, Cypress loads main window in `baseUrl`
1020:### Usage of `baseUrl` in depth

content/api/cypress-api/custom-commands.md
234:#### Log out command using `localStorage` <E2EOnlyBadge />
301:#### Custom `console` command
405:#### Overwrite `visit` command
443:#### Overwrite `type` command
498:#### Overwrite `screenshot` command
521:#### Overwrite `contains` command

content/api/cypress-api/spec.md
17:#### `Cypress.spec` returns an object

content/guides/references/legacy-configuration.md
670:#### <Icon name="angle-right"></Icon> `baseUrl` is not set
702:#### <Icon name="angle-right"></Icon> Test files not found when using `spec` parameter

content/faq/questions/using-cypress-faq.md
242:## <Icon name="angle-right"></Icon> Can I test the HTML `<head>` element?
1103:## <Icon name="angle-right"></Icon> Why doesn't the `instanceof Event` work?
```</details>

Reply:
Cool. I'll revert this one. I checked out a few from this list and it's not an accurate representation. Seems like it's only an issue when used in h3 or h4 headers. Which is most of the list but an FYI when we add a new CSS ticket.

Reply:
Ok good to know! I only checked out a few and, of course, all the ones I looked at had the problem. For sure an easy fix when we get to it.